.github/workflows/e2e-compatibility-unreleased.yaml

```diff
@@ -1,21 +1,48 @@
 name: Compatibility E2E (Unreleased)
-on:
-  workflow_dispatch:
-    inputs:
-      release-branch:
-        description: 'Select unreleased to run all release branches against each other'
-        required: true
-        type: choice
-        options:
-          - unreleased
```

Comment:
this was a legacy from when the workflows were combined, it's not required anymore